### ISSUE TITLE
max/minContains strangely structured paragraphs

### ISSUE THREAD

**Issue by ssilverman** [json-schema-org/json-schema-spec#1044]
https://github.com/json-schema-org/json-schema-spec/blob/97ea4df52ae7d81d1b2d7bd871386e497f7c4cf6/jsonschema-validation.xml#L416-L439

These paragraphs need some punctuation fix-ups, plus they’re hard to follow.

I’ll provide better suggestions soon, I just wanted to capture the issues.

**Comment by Relequestual** [comment]
This was the only way I could think to phrase them. Feel free to have a crack at it.
